src/main/java/ru/training/at/hw6/entities/User.java

```diff
@@ -0,0 +1,32 @@
+package ru.training.at.hw6.entities;
+
+import ru.training.at.hw6.utils.ConfigReader;
+
+public class User {
+
+    public static final User ROMAN = new User(ConfigReader.getProperty("name"),
+            ConfigReader.getProperty("password"),
+            ConfigReader.getProperty("fullName"));
+
+    private String name;
+    private String password;
+    private String fullName;
```

Comment:
I think fields should be final

hint: you can use [lombok Value](https://projectlombok.org/features/Value), if you want